src/univalue/univalue.cpp

```diff
@@ -164,6 +164,19 @@ int UniValue::findKey(const std::string& key) const
     return -1;
 }
 
+const UniValue UniValue::findValue(const std::string& key) const
+{
+    if (typ != VOBJ)
+        return NullUniValue;
```

Comment:
Is it a bug if this happens? I'd say `findValue` shouldn't be called for non-VOBJ in the first place. This is similar to calling get_bool on a non-VBOOL. So maybe it should also raise a std::runtime_error.